### PR TITLE
Better Xml

### PR DESCRIPTION
**Pull request by MaximeZmt**
photo_2022-06-03_01-06-52

### CODE REVIEW

File: app/src/androidTest/java/ch/sdp/vibester/activity/profile/MyProfileFragmentTest.kt [hunk 1]
```diff
@@ -104,6 +104,26 @@ class MyProfileFragmentTest {
     }
 
 
+    @Test
+    fun checkChangePhotoCancel() {
+        val inputProfile = User( "Lalisa Bon","bit.ly/3IUnyAF", "lisa@test.com",  12, 8, "VvPB47tQCLdjz3YebilS6h5EXdJ3")
```
Comment (jiabaow) [inline app/src/androidTest/java/ch/sdp/vibester/activity/profile/MyProfileFragmentTest.kt:109]: can use R.string.test_profile_image

**Review by jiabaow — APPROVED** [review]
LGTM

File: app/src/androidTest/java/ch/sdp/vibester/activity/profile/MyProfileFragmentTest.kt (hunk 1 above)
Comment (MaximeZmt) [reply app/src/androidTest/java/ch/sdp/vibester/activity/profile/MyProfileFragmentTest.kt:109]: I mean that field is useless now, still here for compatibility reasons. I don't think it is worth it to use context for that old field to retrieve a not used value 😅.

**Review by zwierski — APPROVED** [review]
LGTM 👍

**Pull request merged**